src/editor/nodes/ModelNode.js

```diff
@@ -351,26 +349,22 @@ export default class ModelNode extends EditorNodeMixin(Model) {
     return this;
   }
 
-  prepareForExport(ctx) {
+  prepareForExport() {
     super.prepareForExport();
     this.addGLTFComponent("shadow", {
       cast: this.castShadow,
       receive: this.receiveShadow
     });
 
-    // TODO: Support exporting more than one active clip.
-    if (this.activeClip) {
-      const activeClipIndex = ctx.animations.indexOf(this.activeClip);
-
-      if (activeClipIndex === -1) {
-        throw new Error(
-          `Error exporting model "${this.name}" with url "${this._canonicalUrl}". Animation could not be found.`
-        );
-      } else {
-        this.addGLTFComponent("loop-animation", {
-          activeClipIndex: activeClipIndex
-        });
+    this.addGLTFComponent(
+      "loop-animation",
+      {
+        activeClipIndices: this.activeClipIndices
```

Comment:
I think we need to use `this.activeClips.map(clip => ctx.animations.indexOf(clip))` otherwise multiple animated objects will point to the same animation clip. The `ctx.animations` contains all of the animations in the exported glTF scene in the order it will be output.

Reply:
After the naming update this has been replaced with a call to `getActiveClipIndices()` which traverses the animations and return the indices for the currently active clips. Basically what you suggested above.